src/nvim/normal.c

```diff
@@ -6516,25 +6184,21 @@ static void nv_g_cmd(cmdarg_T *cap)
     show_sb_text();
     break;
 
-  /*
-   * "gg": Goto the first line in file.  With a count it goes to
-   * that line number like for "G". -- webb
-   */
+  // "gg": Goto the first line in file.  With a count it goes to
+  // that line number like for "G". -- webb
   case 'g':
     cap->arg = false;
     nv_goto(cap);
     break;
 
-  /*
-   *     Two-character operators:
-   *     "gq"       Format text
-   *     "gw"       Format text and keep cursor position
-   *     "g~"       Toggle the case of the text.
-   *     "gu"       Change text to lower case.
-   *     "gU"       Change text to upper case.
-   *   "g?"         rot13 encoding
-   *   "g@"         call 'operatorfunc'
-   */
+  //    Two-character operators:
+  //    "gq"       Format text
+  //    "gw"       Format text and keep cursor position
+  //    "g~"       Toggle the case of the text.
+  //    "gu"       Change text to lower case.
+  //    "gU"       Change text to upper case.
+  //  "g?"         rot13 encoding
+  //  "g@"         call 'operatorfunc'
```

Comment:
```suggestion
  //    "gu"       Change text to lower case.
  //    "gU"       Change text to upper case.
  //    "g?"       rot13 encoding
  //    "g@"       call 'operatorfunc'
```

i guess we can also fix this while we're at it :P